.claude/docs/lh/lighthouse-reading-guide.md

```
@@ -0,0 +1,252 @@
+- 네트워크: 4G 속도로 강제 제한
+- CPU: 실제보다 4배 느리게 강제 설정
+
+개발 컴퓨터나 WiFi 환경에선 어떤 사이트든 빠르게 느껴지기 때문에 의도적으로 느린 환경을 시뮬레이션한다. 실제 한국 사용자 체감은 이보다 빠를 수 있다.
+
+---
+
+## 2. 브라우저가 페이지를 여는 순서
+
+Lighthouse 지표를 이해하려면 브라우저가 페이지를 여는 순서를 알아야 한다.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)